openslides_backend/action/actions/committee/committee_common_mixin.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, Dict
+
+from ....shared.exceptions import ActionException
+from ...action import Action
+
+
+class CommitteeCommonCreateUpdateMixin(Action):
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        instance = super().update_instance(instance)
+        id = instance.get("id")
+        if id in instance.get("forward_to_committee_ids", []) or id in instance.get(
+            "receive_forwardings_from_committee_ids", []
+        ):
+            raise ActionException(
+                "Forwarding or receiving from own committee is not possible!"
```

Comment:
I think this is wrong: AFAIK, forwarding is possible to the same committee. I guess you misunderstood https://github.com/OpenSlides/openslides-backend/issues/772: The point was that the "forward" and "receive" relations indicated different result, so this must be resolved. One option would be to just set the "forward" field (therefore overwriting the "receive" field), another one to raise an exception because of inconsistent data. This, however, would just move the work to the client, since the user can select its own committee in only one field, which woud have to be caught there, so I'm probably more leaning towards the first solution (if our relation handling is capable of that)

Reply:
Talked with Emanuel and you are right. Changed